src/index.js

```diff
@@ -13,14 +13,24 @@ class DataImport extends React.Component {
     showSettings: PropTypes.bool,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.connectedHome = props.stripes.connect(Home);
+  }
+
   render() {
     if (this.props.showSettings) {
       return <Settings {...this.props} />;
     }
 
     return (
       <Switch>
-        <Route path={`${this.props.match.path}`} exact component={Application} />
+        <Route
+          path={`${this.props.match.path}`}
+          exact
+          render={() => <this.connectedHome {...this.props} />}
```

Comment:
PLease move to additional method and add comments.

Reply:
Will do.